azure/get-akse.sh

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
```

Comment:
Do we really need so much stuff in bash? Maybe just download from Python using `urllib2`, and use `os.system` or `subprocess` to shell out individual commands?

Reply:
I think for this particular script get-akse.sh, we should just use it as it is provided by Msft aks-engine and not try converting into python subprocesses.

Reply:
Then I think it's better not to commit it and just download it on the fly from https://raw.githubusercontent.com/Azure/aks-engine/master/scripts/get-akse.sh, otherwise it will need to be updated and maintained.

Reply:
Ohh I misunderstood you earlier, yes I'll remove the script from here. Thanks!